scriptworker/cot/verify.py

```diff
@@ -1294,18 +1292,17 @@ def compare_jsone_task_definition(parent_link, compare_definition):
         compare_definition = remove_empty_keys(compare_definition)
         runtime_definition = remove_empty_keys(parent_link.task)
 
-        log.debug("Compare_definition:\n{}".format(format_json(compare_definition)))
-        log.debug("Runtime definition:\n{}".format(format_json(runtime_definition)))
         diff = list(dictdiffer.diff(compare_definition, runtime_definition))
         if diff:
-            raise AssertionError(pprint.pformat(diff))
+            error_msg = "{} {}: the rebuilt definition doesn't match the runtime task!\n{}".format(
```

Comment:
Should this be a list? If we end up with multiple tasks in `.taskcluster.yml`, the error is likely going to be misleading.

Or, if we think multiple tasks are going to be unlikely (which is probably the case), we should either
- error out early, if there are multiple tasks
- add a note to the error message about multiple tasks and the diff being against one of them.